.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.com/
```

Comment:
Do we need this config?

Reply:
it is better if we always do that, as many people work on multiple npm registries at once

Reply:
It's redundant in this scenario. as we are always using the default registry `npm`. It would be applicable if we use different registry like `bintray` or `nexus`. So I think we can remove this file.

Reply:
I'm not sure the situation @superhit0 met. I guess maybe he already defined some personal/global `.npmrc` to set his own npm registry server, so need to add the per-project .npmrc config file to make sure it can override them.

Reference: https://docs.npmjs.com/files/npmrc.html#files

Reply:
yes I have a private npm registry. My point was that some one else might come in future as well with a similar situation. 
This doesn't hurt

Reply:
I think your `.npmrc` setup is global. I would be handy if we move our private registry to local.

Reply:
??

Reply:
Still I think it's redundant to have `.npmrc` with default registry path. As we have our `package-lock.json` with default path. If anyone use his own private repo path. it will change the `package-lock` and we can easily review it.

Surely it's not a blocker of this PR. happy to go.

Reply:
> If anyone use his own private repo path. it will change the package-lock and we can easily review it.

I think the lock file is too large to review it, it could be ignored easily for me 😛

---

But, on the other hand, I look at some other libraries, they didn't provide .npmrc in their repos.
Maybe it's not a common way to solve issue like this.

In npm's [doc](https://docs.npmjs.com/configuring-your-registry-settings-as-an-npm-enterprise-user#using-npmrc-to-managing-multiple-profiles-for-different-registries), they mentioned the way to manage multiple registries. Maybe users should do it by themselves to avoid meeting issues, I guess.

---

> Surely it's not a blocker of this PR.

Agree. We can think & revise it in next PR if needed.